Source/WTF/wtf/HashTable.h

```diff
@@ -542,9 +542,9 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(HashTable);
         typedef std::pair<ValueType*, bool> LookupType;
         typedef std::pair<LookupType, unsigned> FullLookupType;
 
-        LookupType lookupForWriting(const Key& key) { return lookupForWriting<IdentityTranslatorType>(key); };
+        ValueType* lookupForReinsert(const Key& key) { return lookupForReinsert<IdentityTranslatorType>(key); };
```

Comment:
Change Log comment says "only checks whether it is an empty." That’s a sentence fragment, missing a word.

Reply:
Changed it to `looks for an insertion point by only checking whether a hashtable entry is empty`.